### PR TITLE
Increase consistency, and style guide compliance

### PR DESCRIPTION
**Pull request by lrakai**
- not using object after API objects per https://kubernetes.io/docs/home/contribute/style-guide/#use-camel-case-for-api-objects

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7503)

### CODE REVIEW

File: docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ Neither contention nor changes to quota will affect already created resources.
...
+Resource quota support is enabled by default for many Kubernetes distributions.  It is
 enabled when the apiserver `--admission-control=` flag has `ResourceQuota` as
 one of its arguments.
 
-Resource Quota is enforced in a particular namespace when there is a
-`ResourceQuota` object in that namespace.  There should be at most one
-`ResourceQuota` object in a namespace.
+A resource quota is enforced in a particular namespace when there is a
+`ResourceQuota` in that namespace.  There should be at most one
+`ResourceQuota` in a namespace.
```
Comment (tengqm) [inline docs/concepts/policy/resource-quotas.md]: Not sure kubernetes does enforce this constraint.

**Comment by steveperry-53** [comment]
@lrakai Thanks for doing this.
/approve

**Comment by steveperry-53** [comment]
Still need an LGTM for technical accuracy. In particular, we need to resolve the question raised by @tengqm.

**Comment by lrakai** [comment]
Fwiw, a [different commit](https://github.com/kubernetes/website/commit/8d7accb1a947978282b69794b70c608cc094ea35#diff-94ddc6e901c30f256138db11d09f05a3L50) removed the line that I believe was in question.  I merged it in with my changes.

**Comment by steveperry-53** [comment]
@lrakai Sounds good to me. Just to follow procedure, I'll let @tengqm apply the LGTM.

**Pull request merged**